src/api/libcellml/validator.h

```diff
@@ -53,7 +53,7 @@ class LIBCELLML_EXPORT Validator: public Logger
      * @brief Validate the @p model using the CellML 2.0 Specification.
      *
      * Validate the given @p model and its encapsulated entities using the
-     * CellML 2.0 Specification. Any errors will be logged in the @c Validator.
```

Comment:
The validator only reports errors so it should be errors.

Reply:
Well spotted. Done in commit 5f478db0.